doc/developers/contributing.rst

```diff
@@ -224,39 +227,47 @@ then submit a "pull request" (PR):
 
     $ git remote add upstream https://github.com/scikit-learn/scikit-learn.git
 
-7. Fetch the ``upstream`` and then create a branch to hold your development
-   changes::
+You should now have a working installation of scikit-learn, and your git
+repository properly configured. The next steps now describe the process of
+modifying code and submitting a PR:
+
+7. Synchronize your master branch with the upstream master branch::
+
+        $ git checkout master
+        $ git fetch upstream
```

Comment:
Then git merge or rebase maybe? Otherwise the master is not synced with upstream

Reply:
right

would you be ok with `git pull`? that's what I use

Reply:
:thinking: not sure how to sync with git pull, we seem to all do it in different ways. I'm happy with whatever works :)